docs/zap_clusters.md

```diff
@@ -127,6 +127,7 @@ Generally regenerate using one of:
 |       1295 |      0x50F | ContentControl                                          |
 |       1296 |      0x510 | ContentAppObserver                                      |
 |       2820 |      0xB04 | ElectricalMeasurement                                   |
+|       3072 |      0xC00 | WiFiNetworkManagement                                   |
```

Comment:
This does not match the cluster ID I see in the spec?  That seems to be 0x0451.

Reply:
Addressed in the XML below (this one is generated)